### PR TITLE
Include component icons as library assets

### PR DESCRIPTION
**Pull request by Arlodotexe**
Needs https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/129

This PR consists of a tooling update that fixes an asset issue in the sample app.

Component icons and other `.png` files are now included as library assets instead of using the `/SourceAssets` workaround needed for this (now closed) issue: https://github.com/unoplatform/uno/issues/2502.

### CODE REVIEW

**Comment by michael-hawker** [comment]
Oh, wait this one is still broken and appears to be looking in the wrong place:

image

HeaderedContentControl doc... Miss this project maybe @Arlodotexe?

File: components/HeaderedControls/samples/HeaderedControls.Samples.csproj
Comment (michael-hawker) [inline components/HeaderedControls/samples/HeaderedControls.Samples.csproj:1]: Not sure why, but saw issues with the sample code/xaml here on this one specifically, and it was still calling out SourceAssets?

Comment (Arlodotexe) [reply components/HeaderedControls/samples/HeaderedControls.Samples.csproj:1]: This PR only touches on png files. Markdown, Xaml and cs file handling are untouched.

**Review by michael-hawker — APPROVED** [review]
@Arlodotexe looks good then, approving so you'll be all set once the Tooling PR merges to update the pointer here and get this merged.

**Pull request merged**
